### ISSUE TITLE
Problem with connecting to cisco IOS XE.

### ISSUE THREAD

**Issue by jonathansaha** [hsdn/lg#35]
Hello,

I am facing an issue which has some similarities to "SSH and Image_GraphViz problems", but also quite different.

I setup a server with the front end, just followed this github's readme, example public telnet looking glasses working fine. I also tried with other public telnet looking glasses, they all worked fine. Please note that I only checked with cisco IOS XE varient, even if I dont mention later, assume that I only tested/tried against IOS XE.

Problem starts when I try to attach my own router, its a cisco A901. It shows following error.
![image](https://user-images.githubusercontent.com/14842609/108079903-85add000-7099-11eb-8508-afb467e62ba7.png)

First I thought it was not playing well with ssh, so I changed it to telnet. Then I thought may be my password having a "@" character causing the issue. So I removed the password too. Currently it is telnet with no password. Error is same as above.

Public telnet looking glass working fine when attached. Also when I try from the server cli, I can reach my telnet with no issue, just like public ones.

This one is mine.
![image](https://user-images.githubusercontent.com/14842609/108087077-45eae680-70a1-11eb-88c6-0e786a753e80.png)

This one is a public one from this github's example, which also works fine in front end.
![image](https://user-images.githubusercontent.com/14842609/108081368-1802a380-709b-11eb-96f6-46e78a6e3d5f.png)

So from server cli, same server hosting the front end, I can interact with both looking glasses same way without any issue. Note that router commands for these two routers, public example and mine, are identical.

Below is my lg_config.php, except for "example1" paragraph, rest is default. I tried with port number at the end and without.
![image](https://user-images.githubusercontent.com/14842609/108082003-d0c8e280-709b-11eb-8270-17c97e244f5a.png)

Please note that first I tried without a hostname, just a direct ip address, I thought it might cause an issue so I changed the ip to hostname at /etc/hosts.

Also note that since I have not been able to got this to production, and my company, and me, has a telnet phobia, my telnet looking glass has private IP to login. But as long as its reachable from the server, it shouldn't be an issue. Still mentioned it just in case.

Server runs on Ubuntu 20.04.1 LTS, my router is cisco A901 with IOS XR.

long story short, front end works fine. Back end telnet looking glass also works fine. But when I try to attach them I get the error from the first image. I tried to copy my looking glass as much as like the public one in this example. I dont know about the public one's internal, but from putty or server cli, connection process and commands are identical.

http://103.131.159.232/ if you want to see for yourself.

Thanks for taking your time to read the post. Bgp graph feature is really neat.
